src/server/scripts/EasternKingdoms/zone_undercity.cpp

```diff
@@ -126,10 +116,23 @@ class npc_lady_sylvanas_windrunner : public CreatureScript
         uint32 BlackArrowTimer;
         uint32 ShotTimer;
         uint32 MultiShotTimer;
+        bool flag;
+        Player * _player;
```

Comment:
I understand why this would typically be a bad thing to do; however only one person can trigger this script at a time so what's the problem?